### DISCUSSION TITLE
✨ Lemmy

### DISCUSSION THREAD

**Discussion by tempofunktron** [runtipi/runtipi-appstore#2826] [Requests]
### Description

Link aggregator (like Reddit) built on ActivityPub for the fediverse.

### Official Website

https://join-lemmy.org/

### Documentation link

https://join-lemmy.org/docs/

### Last application release & date

0.19.3 (22/01/24)

### Application license

AGPL-3.0 license

### Source code repository link

https://github.com/LemmyNet/lemmy

### Docker image link

https://github.com/LemmyNet/lemmy/tree/main/docker

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote ⬆️ this discussion to show your interest in this request! 

Thanks ⛺

**Comment by dilroopg** [comment]
would be great to see support, only yunohost supports it rnow, cosmosclouds is outdated.
